pkg/runtime/cli.go

```diff
@@ -92,7 +93,8 @@ func FromFlags() (*DaprRuntime, error) {
 	metricsExporter.Options().AttachCmdFlags(flag.StringVar, flag.BoolVar)
 
 	// Finally parse the CLI flags!
-	flag.Parse()
+	// Ignore errors; CommandLine is set for ExitOnError.
+	_ = flag.CommandLine.Parse(args)
```

Comment:
Can you move this to a separate PR? I can see how this could be useful but it's probably worth a separate, deeper conversation.

Reply:
Sure, I'll move it to a separate pr.

Reply:
Moved to #6372